rtmp-services: Add NFHS Network

migillett opened the pull request:
### Description
Added rtmp service for the [NFHS Network](https://www.nfhsnetwork.com/).

We modified the `package.json` and `services.json` files inside of the `rtmp-services` plugin to include the primary RTMP endpoint for the NFHS Network. The changes to the plugin also include our supported resolutions and maximum frame rate.

### Motivation and Context
Our primary motivation for this was to simplify the process of getting our customers streaming to our service and prevent potential user errors when re-typing or pasting the RTMP URL.

### How Has This Been Tested?
- JSON Validation
- Built using cmake and compiled using XCode on a 2021 Macbook Pro (Apple M1 Pro) running MacOS Sonoma 14.4.1
- Performed a test stream to our service using the new NFHS Network service dropdown option
- Verified RTMP stream displayed on the service

### Types of changes
- New feature (non-breaking change which adds functionality)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

norihiro commented:
The first word should be capitalized. `add` --> `Add`.

migillett commented:
@norihiro Apologies, updated name of PR to `rtmp-services: Add NFHS Network`

Fenrirthviti reviewed (approved):
Approved from an eligibility standpoint.

norihiro commented:
> @norihiro Apologies, updated name of PR to `rtmp-services: Add NFHS Network`

The commit message has not been fixed.
You need to run `git commit --amend` and edit the commit message. Then, you need to force-push the commit to GitHub.

migillett commented:
@norihiro And fixed! Apologies, first time contributing to the project.

RytoEX commented:
Unfortunately, with the merge of #11080, this will need to be rebased to correctly increment the versions in package.json.

migillett commented:
I fixed the merge conflict and bumped the rtmp services version from 260 (master) to 261. @RytoEX

RytoEX reviewed (changes requested):
Do not use merge commits to fast forward history. Use `git rebase`.

migillett commented:
Sounds good, I'm on it @RytoEX. I'll ping you when it's ready.

migillett commented:
@RytoEX Ok, I think that did it. Rebased to remote master and squashed all of my commits into one. Also made sure that the commit message was correct like specified earlier.

RytoEX commented on plugins/rtmp-services/data/package.json:
```diff
@@ -1,11 +1,11 @@
         }
     ]
-}
+}
```
Revert this change. Always end the file with a newline.

migillett replied:
Fixed!
